### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by AnastasiiaYaitska**

### CODE REVIEW

File: javascript/javascript1/week1/task-1.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Age-ify (A future age calculator)
+
+// 1
+
+const yearOfBirth = 1991;
+const yearFuture = 2023;
+const age = yearFuture - yearOfBirth;
+console.log(`You will be ${age} years old in ${yearFuture}`);
```
Comment (MaherHussain) [inline javascript/javascript1/week1/task-1.js:8]: good job with using literals in console.  :-)

File: javascript/javascript1/week1/task-3.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+let houseHigh = prompt('hose higt?');
+const volumeInMeters = houseDeep * houseHigh * houseWidth;
+let gardenSizeInM2 = prompt('garden size in m2?');
+let housePrice = prompt('house price?');
+
+if (housePrice >= volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300) {
+   console.log("You pay too much!");
+} else {
+console.log('It is a good price =)');
+};
```
Comment (MaherHussain) [inline javascript/javascript1/week1/task-3.js:15]: good using for prompt method, but it's better to follow the homework requirements  
always any user inputs need validation and check type, and here in your case, the first condition will never be true if the user adds strings in the input field..,

**Comment by MaherHussain** [comment]
great job. feel free to merge your PR

File: javascript/javascript1/week1/task-3.js (hunk 2 above)
Comment (AnastasiiaYaitska) [reply javascript/javascript1/week1/task-3.js:15]: Thank you! I get some ideas, maybe I have to use objects instead of it?

**Pull request merged**